Source/WebCore/Modules/fetch/FetchBodyConsumer.cpp

```diff
@@ -385,6 +383,7 @@ void FetchBodyConsumer::resolve(Ref<DeferredPromise>&& promise, const String& co
         promise->resolve<IDLDOMString>(takeAsText());
         return;
     case FetchBodyConsumer::Type::FormData: {
+        auto* context = promise->scriptExecutionContext();
```

Comment:
It is only used once, let's call packageFormData with `promise->scriptExecutionContext()` directly